api/proto/teleport/autoupdate/v1/autoupdate.proto

```diff
@@ -51,5 +70,14 @@ message AutoUpdateVersion {
 // AutoUpdateVersionSpec encodes the parameters of the autoupdate versions.
 message AutoUpdateVersionSpec {
   // ToolsVersion is the semantic version required for tools autoupdates.
+  // Deprecated: use AutoUpdateVersionTools instead.
   string tools_version = 1;
+
+  AutoUpdateVersionTools tools = 2;
+}
+
+// AutoUpdateVersionTools encodes the parameters for client tools auto updates.
+message AutoUpdateVersionTools {
+  // TargetVersion is the semantic version required for tools autoupdates.
```

Comment:
Is this the target version we want you to upgrade to or is it the version required for you to be able to update to something else?

Reply:
@vapopov what's the correct version here?

Reply:
@tigrato TargetVersion is mandatory version for update, if `mode` is `enabled` so everyone who is connected to the cluster going to be updated to this version (except client tools executed with disabling env variable)